ch_backup/logic/table.py

```diff
@@ -605,6 +612,7 @@ def _restore_data(
                                 context.backup_meta, part, fs_part_path
                             )
 
+                        context.restore_context.add_part(part, PartState.DOWNLOADED)
```

Comment:
Actually,  the part has been not downloaded yet. Because `download_data_part()` downloads parts asynchronously. All parts will be downloaded after `context.backup_layout.wait(keep_going)`.

Reply:
Oh, forgot about it, I'm fixing redundant copying on restart for cloud storage